src/skvhttpclient/src/skvhttp/common/Common.h

```diff
@@ -32,18 +32,11 @@ Copyright(c) 2020 Futurewei Cloud
 #include <skvhttp/common/Serialization.h>
 
 #include <boost/thread/future.hpp>
-
-
-#ifdef _GLIBCXX_USE_DECIMAL_FLOAT
-#include <decimal/decimal>
 namespace skv::http {
-using Decimal64 = std::decimal::decimal64;
-using Decimal128 = std::decimal::decimal128;
-auto constexpr Decimal64_to_float = std::decimal::decimal64_to_float;
-auto constexpr Decimal128_to_float = std::decimal::decimal128_to_float;
+using DecimalD25 = boost::multiprecision::cpp_dec_float_25;
+using DecimalD50 = boost::multiprecision::cpp_dec_float_50;
+using DecimalD100 = boost::multiprecision::cpp_dec_float_100;
```

Comment:
Why not use these in the entire codebase? I think it would read better everywhere instead of the fully specified 'boost::multiprecision::cpp_dec_float"

Reply:
fixed